### ISSUE TITLE
NameError: name 'self' is not defined in adapter_hpfanficarchivecom.py

### ISSUE THREAD

**Issue by Octav14n** [JimmXinu/FanFicFare#549]
Executing "fanficfare -s" gives the following error after updating:
(It should show all supported domains with url-examples)

```
$ fanficfare -s
Traceback (most recent call last):
  File "/home/bofilo/venv/bin/fanficfare", line 8, in <module>
    sys.exit(main())
  File "/home/bofilo/venv/lib/python3.6/site-packages/fanficfare/cli.py", line 228, in main
    for site, examples in adapters.getSiteExamples():
  File "/home/bofilo/venv/lib/python3.6/site-packages/fanficfare/adapters/__init__.py", line 272, in getSiteExamples
    l.append((cls.getConfigSection(),cls.getSiteExampleURLs().split()))
  File "/home/bofilo/venv/lib/python3.6/site-packages/fanficfare/adapters/adapter_hpfanficarchivecom.py", line 70, in getSiteExampleURLs
    return self.protocol+cls.getSiteDomain()+"/stories/viewstory.php?sid=1234"
NameError: name 'self' is not defined
```

**Comment by JimmXinu** [comment]
Yeah, I noticed that about a minute after releasing. :-(

It's fixed in c57470e95571322cae054f334a9256dc530c5687.

Is `-s` an option you actually _use_ for something?  And if so, what?  Because I'm debating about the value putting out a point release to fix it or not.

**Comment by Octav14n** [comment]
I am using it in my android app wrapper [BoFiLo](https://github.com/Octav14n/BoFiLo), I have downgraded at the moment.

**Comment by JimmXinu** [comment]
Released CLI only v3.23.1.
